ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -35,8 +35,9 @@ finished()
   echo "dev-scripts setup completed, fetching logs"
   ssh "${SSHOPTS[@]}" "root@${IP}" tar -czf - /root/dev-scripts/logs | tar -C "${ARTIFACT_DIR}" -xzf -
   echo "Removing REDACTED info from log..."
+  # Use 's/auths.*$/' instead of 's/.*auths.*/' to avoid regex backtracking on long lines
   sed -i '
-    s/.*auths.*/*** PULL_SECRET ***/g;
+    s/auths.*$/*** PULL_SECRET ***/;
```

Comment:
Previously this redacted the whole line, but now only the bit starting with `auths`. I think an efficient equivalent to the previous command would be:

```
/auths/ s/.*/*** PULL_SECRET ***/
```

The whole thing is weird though, because a pull secret contains `auth` not `auths`.